requires @types/command-line-usage

I find that in order to compile a project using this package, I have to install @types/command-line-usage myself as a dev dependency. Otherwise I get this:

```
node_modules/ts-command-line-args/dist/helpers/options.helper.d.ts:1:37 - error TS7016: Could not find a declaration file for module 'command-line-usage'. '/Users/geof.holbrook/Data/repos/bridge/node_modules/command-line-usage/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/command-line-usage` if it exists or add a new declaration (.d.ts) file containing `declare module 'command-line-usage';`

1 import { Content, OptionList } from 'command-line-usage';
```

--- a/src/contracts.ts
+++ b/src/contracts.ts
@@ -313,3 +313,17 @@ export type UsageGuideConfig<T = any> = {
     arguments: ArgumentConfig<T>;
     parseOptions?: ParseOptions<T>;
 };
+
+export interface OptionList {
+    header?: string;
+    /** An array of option definition objects. */
+    optionList?: OptionDefinition[];
+    /** If specified, only options from this particular group will be printed.  */
+    group?: string | string[];
+    /** The names of one of more option definitions to hide from the option list.  */
+    hide?: string | string[];
+    /** If true, the option alias will be displayed after the name, i.e. --verbose, -v instead of -v, --verbose). */
+    reverseNameOrder?: boolean;
+    /** An options object suitable for passing into table-layout. */
+    tableOptions?: any;
+}

--- a/src/helpers/options.helper.ts
+++ b/src/helpers/options.helper.ts
@@ -1,5 +1,12 @@
-import { Content, OptionList } from 'command-line-usage';
-import { ParseOptions, OptionContent, CommandLineOption, OptionalProperty, PropertyOptions } from '../contracts';
+import {
+    ParseOptions,
+    OptionContent,
+    CommandLineOption,
+    OptionalProperty,
+    PropertyOptions,
+    Content,
+    OptionList,
+} from '../contracts';
 
 export function getOptionSections(options: ParseOptions<any>): OptionContent[] {
     return (

--- a/src/parse.spec.ts
+++ b/src/parse.spec.ts
@@ -14,7 +14,6 @@ import { parse } from './parse';
 import * as fsImport from 'fs';
 import * as pathImport from 'path';
 import * as helpersImport from './helpers';
-import { ParseOptions } from '.';
 
 jest.mock('fs', () => require('@morgan-stanley/ts-mocking-bird').proxyJestModule(require.resolve('fs')));
 jest.mock('path', () => require('@morgan-stanley/ts-mocking-bird').proxyJestModule(require.resolve('path')));

